digdag-cli/src/main/java/io/digdag/cli/client/Backfill.java

```diff
@@ -63,23 +82,29 @@ public SystemExitException usage(String error)
         return systemExit(error);
     }
 
-    private void backfill(String projName, String workflowName)
+    private void backfillScheduleId(Id scheduleId)
         throws Exception
     {
-        LocalTimeOrInstant fromTime = LocalTimeOrInstant.of(
-                    TimeUtil.parseLocalTime(fromTimeString,
-                        "--from must be hourly, daily, now, \"yyyy-MM-dd\", or \"yyyy-MM-dd HH:mm:SS\" format"));
-
         DigdagClient client = buildClient();
+        RestSchedule sched = client.getSchedule(scheduleId);
+        backfill(sched, client);
+    }
 
-        RestSchedule sched = findScheduleByWorkflowName(client, projName, workflowName);
+    private void backfillWorkflow(String projectName, String workflowName)
+        throws Exception
+    {
+        DigdagClient client = buildClient();
+        RestProject project = client.getProject(projectName);
+        RestSchedule sched = client.getSchedule(project.getId(), workflowName);
+        backfill(sched, client);
+    }
 
-        if (sched == null) {
-            // confirm that project and workflow exist, otherwise throws an exception
-            RestProject proj = client.getProject(projName);
-            RestWorkflowDefinition def = client.getWorkflowDefinition(proj.getId(), workflowName);
-            throw systemExit("Schedule is not set to the workflow");
-        }
+    private void backfill(RestSchedule sched, DigdagClient client)
+        throws Exception
+    {
+        LocalTimeOrInstant fromTime = LocalTimeOrInstant.of(
+                    TimeUtil.parseLocalTime(fromTimeString,
+                        "--from must be \"yyyy-MM-dd\", or \"yyyy-MM-dd HH:mm:SS\" format"));
```

Comment:
Also, fixed a wrong error message

```
--from must be hourly, daily, now, \"yyyy-MM-dd\", or \"yyyy-MM-dd HH:mm:SS\" format"
```

that hourly, daily, now are not supported. Please let me know if you prefer me to send separated PR.